packages/graphql/lib/src/core/query_options.dart

```diff
@@ -92,8 +92,8 @@ class QueryOptions extends BaseOptions {
   QueryOptions({
     @required String document,
     Map<String, dynamic> variables,
-    FetchPolicy fetchPolicy = FetchPolicy.cacheFirst,
-    ErrorPolicy errorPolicy = ErrorPolicy.none,
+    FetchPolicy fetchPolicy,
```

Comment:
Why take out these defaults?

Reply:
Defaults are now configurable in `GraphQLClient.defaultPolicies` (#398), so they need to be `null` to be overridden later. So this is actually more of a fix 😅